Library.py

```diff
@@ -0,0 +1,161 @@
+        if len(books) > 0:
+            book = input("Which book you want to borrow: ")
+            for i in self.database["books"]:
+                if i["title"] == book:
+                    i["borrow"] = "Yes"
+                    with open("database.json", "w") as f:
+                        f.write(json.dumps(self.database))
+                    self.database['registry'].append({'user': user, 'book': book})
+                    with open("database.json", "w") as f:
+                        f.write(json.dumps(self.database))
+                    print(f"{user} just borrow book: {book}")
+        else:
+            print("Library is empty, 'admin' should deliver new books or wait maybe some user will return book :) ")
+
+    def available_books_to_deposit(self, user, books):
+        print("Books available to deposit:")
+        for i in self.database["registry"]:
+            if i["user"] == user:
+                print(i["book"])
+                books.append(i["book"])
+            else:
+                pass
+
+    def deposit(self, user):
+        books = []
+        self.available_books_to_deposit(user, books)
+        if len(books) > 0:
+            book = input("Which book you want to return: ")
+            if book in books:
+                for i in self.database["registry"]:
+                    if i["book"] == book:
+                        self.database["registry"].remove(i)
+                for b in self.database["books"]:
+                    if b["title"] == book:
+                        b["borrow"] = "No"
+                with open("database.json", "w") as f:
+                    f.write(json.dumps(self.database))
+                    print(f"{user} just deposit book: {book}")
+            else:
+                print(f"You don't have book: {book}")
+                book = input("Which book you want to return: ")
+                self.deposit(user)
+        else:
+            print("You don't have any book, please borrow some first :) ")
+
+    def user_log_in(self, login, password):
+        if login != "" and password != "":
+            for i in self.database["users"]:
+                if login == i["login"] and password == i["password"]:
+                    return True
+            return False
+
+    def login_available(self, user_name):
+        for i in self.database["users"]:
+            if i["login"] == user_name:
+                return False
+        return True
+
+    def password_available(self, user_password):
+        for i in self.database["users"]:
+            if i["password"] == user_password:
+                return False
+        return True
+
+    def register_user(self, user_name, user_password):
+        if self.login_available(user_name) and self.password_available(user_password):
+            self.database["users"].append({"login": user_name, "password": user_password})
+            with open("database.json", "w") as f:
+                f.write(json.dumps(self.database))
```

Comment:
wielokrotnie powtarza się ta funkcja w kodzie, zamknij to w osobnej metodzie i wywołuj gdzie trzeba